spec/models/service_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Service, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
If we do add validations or associations then we can put something in these specs to cover them.  Consider using the [`shoulda-matchers` gem](https://github.com/thoughtbot/shoulda-matchers) for testing that sort of thing.  If you've never used it there are a bunch of other gds apps that use it so we can crib from them, but the docs are pretty good starting point too.